Sources/Tools/plPythonPack/main.cpp

```diff
@@ -61,6 +61,15 @@ static const std::unordered_set<ST::string, ST::hash_i, ST::equal_i> s_ignoreSub
     ST_LITERAL("__pycache__"),
 };
 
+/** Modules that should never be packed. */
+static const std::unordered_set<ST::string, ST::hash_i, ST::equal_i> s_ignoreModules{
+    // These built-in modules that are part of Python itself.
+    // Do NOT try to import them... it will give Python heartburn,
+    // which is basically fatal since we share the interpreter state.
+    ST_LITERAL("importlib._bootstrap"),
+    ST_LITERAL("importlib._bootstrap_external"),
+};
```

Comment:
Thanks - I think I ran into this a while ago, but never got around to doing anything about it.